lib/llm-events/utils.js

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
Note: This is the same file as this in PR:https://github.com/newrelic/node-newrelic/pull/3430/files#diff-d2cde635cbccfcbf23c4e9a87a19ee85e1e80ce57198a7284fc837cfe86975f9 so I'll rebase this PR once that other PR is merged.